kolibri/plugins/coach/assets/src/views/groups-page/index.vue

```diff
@@ -10,22 +10,22 @@
     />
 
     <create-group-modal v-if="showCreateGroupModal"
-      :groups="sortedGroups" />
```

Comment:
Is this for the 'no-multi-spaces' rule?

It would be nice if there were a rule to put the last bracket on a newline, like there is for the react eslint plugin.

Reply:
This removal of whitespace was actually done manually by me. 

> It would be nice if there were a rule to put the last bracket on a newline, like there is for the react eslint plugin.

Might be address this using `html-indent` and `max-attributes-per-line`

Reply:
Yeah, if those rules don't work, this should have its own rule. Maybe a Hacktoberfest project?